### PR TITLE
Print styles

### PR DESCRIPTION
**Pull request by ztolley**
* Added a print stylesheet
* Instructed rails to build the print.css asset
* Updated the top level layout to use the print.css when needed
* Created styles to avoid bad page breaks and hide things that don't need to be seen.

screen2
screen1

### CODE REVIEW

**Comment by rgarner** [comment]
There's still a "Continue" button at the bottom of the printed sheet – should we be removing this?

screen shot 2017-07-31 at 10 11 56

**Comment by ztolley** [comment]
You should not be seeing that button, that's weird.. didn't do it on mine.

**Comment by ztolley** [comment]
In fact Russel it's not applying any of the styles at all.

**Comment by rgarner** [comment]
Server restart has fixed it. Please consider that comment void :smile:

**Comment by rgarner** [comment]
LGTM

File: app/assets/images/printer-tool.svg [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+			v121.302c0,16.764,13.595,30.326,30.327,30.326h242.604c16.765,0,30.327-13.562,30.327-30.326V333.584h60.653
+			c16.759,0,30.322-13.563,30.322-30.327V151.628C485.212,134.895,471.648,121.304,454.89,121.304z M363.909,454.886H121.305
+			v-212.28h242.604V454.886z M424.562,212.282c-16.764,0-30.326-13.564-30.326-30.327c0-16.731,13.562-30.327,30.326-30.327
+			c16.765,0,30.327,13.595,30.327,30.327C454.89,198.717,441.327,212.282,424.562,212.282z"/>
+		<rect x="151.629" y="272.93" width="121.304" height="30.327"/>
+		<rect x="151.629" y="333.584" width="181.958" height="30.321"/>
+		<rect x="151.629" y="394.237" width="181.958" height="30.322"/>
+	</g>
+</g>
+</svg>
```
Comment (rgarner) [inline app/assets/images/printer-tool.svg]: Add a comment to start a review which I will immediately approve

**Pull request merged**